src/iexamine.cpp

```diff
@@ -1793,8 +1793,26 @@ void iexamine::locked_object( Character &you, const tripoint &examp )
     map &here = get_map();
     item &best_prying = you.item_with_best_of_quality( qual_PRY, true );
     item &best_lockpick = you.item_with_best_of_quality( qual_LOCKPICK, true );
+    cata::optional<vpart_position> veh = here.veh_at( examp );
+    int locked_part = -1;
+    std::string part_name;
```

Comment:
Making this variable save whatever you're lockpicking (part/furniture/terrain name) here would probably save the 5 nested ternaries below

Reply:
Done.